addons/godot-xr-tools/functions/Function_Climb_movement.gd

```diff
@@ -122,11 +122,13 @@ func _set_climbing(active: bool, player_body: PlayerBody) -> void:
 	if is_active:
 		_distances.clear()
 		_deltas.clear()
+		player_body.override_player_height("climb", 0.0)
```

Comment:
here you hardcode a string "climb" while in crouch you use self. We should chose either one or the other and be consistent

Reply:
I made the override_player_height take a variant so it could be any type of key. Originally I was just using strings, but then I found that I wanted the left and right controller crouch buttons to both submit separate crouch requests which is why I made it take a variant and just used the nodes. You are correct however that in this case using 'self' may actually be cleaner - in debugging it would then be possible to find out which object is requesting the height adjustment and actually point back to the node. 

I'll push up the change.